backend/src/main/java/com/bakdata/conquery/mode/local/LocalStorageListener.java

```diff
@@ -1,7 +1,40 @@
 package com.bakdata.conquery.mode.local;
 
 import com.bakdata.conquery.mode.StorageListener;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.datasets.SecondaryIdDescription;
+import com.bakdata.conquery.models.datasets.Table;
+import com.bakdata.conquery.models.datasets.concepts.Concept;
 
 public class LocalStorageListener implements StorageListener {
-	// Nothing to overwrite because there are no additional operations required when running in local mode
+
+	// When running without shards, no further actions are required
+
+	@Override
+	public void onAddSecondaryId(SecondaryIdDescription secondaryId) {
+	}
+
+	@Override
+	public void onDeleteSecondaryId(SecondaryIdDescription description) {
+	}
+
+	@Override
+	public void onAddTable(Table table) {
```

Comment:
Das könnte man später gut benutzen, um Valdierung zu machen, also wenn eine Tabelle angelegt wird, testen ob sie in SQL existiert mit den angegebenen Spalten.